### PR TITLE
Update site with new content and dates

### PR DESCRIPTION
**Pull request by ironng**
julykeepwebsiteupdates

For #47, #69, #66, #50, #6, #71   

- Added Luis Cuende Quote and new blog content section
- Added Joe Urgo Quote and "We're Hiring" section
- Updated timeline with new copy and fixed spacing issues
- Addressed word wrapping issue in timeline labels on mobile
- Replaced dependency on bootstrap css cdn with npm package
- Updated primer to use latest version

### CODE REVIEW

File: src/BlogContentSection.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        super(props)
+
+        this.state = {
+            blogItems: []
+        }
+    }
+
+    componentDidMount() {
+        request
+            .get('https://api.rss2json.com/v1/api.json')
```
Comment (mhluongo) [inline src/BlogContentSection.jsx]: I want to replace this with our own backend desperately. What's the worst case if they return malicious data? If it's anything but an ugly page let's see about self hosting or putting up our own endpoint... Or even doing it entirely client-side

Comment (ironng) [reply src/BlogContentSection.jsx]: Good call out. I'm not sure what security measures https://rss2json.com/docs takes, if any, if malicious data is sent. I couldn't find any information, and I don't think the code is open source - I checked the developer's github and there's no repo for this code. Sorry for not thinking about security!

We can do any of the options you mentioned, but what do you want to do in the mean time? I can either remove the blog content section or just hard code this content temporarily until we have the endpoint/self-hosting ready.

Comment (mhluongo) [reply src/BlogContentSection.jsx]: Let's hard code it in the meantime

**Comment by ironng** [comment]
This is ready for re-review. Note that the timeline is now completely removed, as per the latest comments in #66.

**Comment by Shadowfiend** [comment]
Ok, let's rock and roll.

**Pull request merged**